addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/dialogs/ZestRecordScriptDialog.java

```diff
@@ -176,6 +194,21 @@ private List<String> getSites() {
         return list;
     }
 
+    private void launchBrowser(String url, String browserName) {
+        ExtensionSelenium extSelenium =
+                Control.getSingleton().getExtensionLoader().getExtension(ExtensionSelenium.class);
+        try {
+            WebDriver wd = extSelenium.getProxiedBrowserByName(browserName);
+            wd.get(url);
+        } catch (RuntimeException e) {
+            String msg =
+                    extSelenium.getWarnMessageFailedToStart(
+                            browserName.toLowerCase(), new Throwable(e));
```

Comment:
- The lower case should use `Locale.ROOT`.
- Pass just `e`.

Reply:
I am curious about what does Locale.Root does

Reply:
It's a neutral locale to change the case of the characters, in some locales changing the case might change the character/letter.

Reply:
But I think this chrome, firefox value is not i18nised. These are the id of the browser, does it still need this?

Reply:
It does not matter if it's i18n'ed, the change happens regardless. Yes, it's still better to use a neutral locale than rely on the default.

Reply:
Okay got it. Thank you.